submissions/dmytro-botnarenko/html-css-popup/style.css

```diff
@@ -0,0 +1,153 @@
+*,
+*:before,
+*:after {
+  padding: 0;
+  margin: 0;
+  border: 0;
+  box-sizing: border-box;
+}
+html,
+body {
+  height: 100%;
+  background-color: white;
+  font-family: Tahoma, Helvetica, Arial, sans-serif;
+  color: #fff;
+  font-size: calc(12px + 0.390625vw);
+  line-height: 24px;
+  letter-spacing: 0.05em;
+}
+a {
+  color: #794f45;
+  text-decoration: none;
+}
+.header {
+  padding: 12px 14px;
+  box-shadow: 0px 1px 2px 1px rgba(0, 0, 0, 0.2);
+}
+.wrapper {
+  min-height: 100%;
```

Comment:
why this property needed?

Reply:
At the beginning of the project I needed this block, but now I realized that it seems that this `div .wrapper` is not needed at all. Removed in HTML and CSS.